services/invoice/server/generatePdfService.ts

```diff
@@ -21,86 +17,95 @@ import { InvoiceType } from "@/types";
  * @returns {Promise<NextResponse>} A promise that resolves to a NextResponse object containing the generated PDF.
  */
 export async function generatePdfService(req: NextRequest) {
-	const body: InvoiceType = await req.json();
-	let browser;
-	let page;
+    const body: InvoiceType = await req.json();
+    let browser;
+    let page;
```

Comment:
try looking at the diff without whitespace changes here:

- https://github.com/al1abb/invoify/pull/747/files?w=1